app/views/overview/_traffic-percent.html

```diff
@@ -0,0 +1,17 @@
+<div ng-if="alternateServices.length">
+  <span class="visible-xs visible-sm">
+    Traffic {{(weightByService[service.metadata.name] / totalWeight) | percent}}
```

Comment:
probably a chance of us having a divide by zero here once we get them to allow zero for weight, unless we require at least one non-zero weight